### DISCUSSION TITLE
Looking for contribution

### DISCUSSION THREAD

**Discussion by alidotdeveloper** [CropperBlazor/Cropper.Blazor#223] [General]
Hey everyone, I'm looking for the contribution in that project. is this project is finished, I didn't find much issues now. Thanks!

**Comment by MaxymGorn** [comment]
Hi @alidotdeveloper . So, actually the project is completed, except that we are currently using version 1.6.0 of cropper.js, and there is already version 2, which is still in beta development. 

Secondly, with the release of .net 8, it will also be supported in this project.

Moreover, for data contracts not all links are displayed correctly in demo site: https://cropperblazor.github.io/contract/Options#properties
![Screenshot_20230922_010313_Chrome.jpg](https://github.com/CropperBlazor/Cropper.Blazor/assets/50423072/16427435-1f03-40cd-b237-37b29a6f343e)

**Comment by ivg-steve** [comment]
Thank you for making this project available!. Is it possible to get the code for your demo site, please?
thanks in advance.

**Reply by MaxymGorn** [reply]
Sure, behold: https://github.com/CropperBlazor/Cropper.Blazor/tree/dev/src%2FCropper.Blazor%2FClient. Actually, I'm incorporating all links that were not displayed properly.

**Comment by ivg-steve** [comment]
Thanks for the quick response!
